backend/src/main/java/de/bund/digitalservice/ris/norms/domain/entity/Proprietary.java

```diff
@@ -225,4 +225,18 @@ public Optional<String> getOrganisationsEinheit(final LocalDate date) {
     return getMetadatenDs()
         .flatMap(m -> m.getSimpleMetadatum(MetadatenDs.Metadata.ORGANISATIONS_EINHEIT, date));
   }
+
+  /**
+   * Returns the ("Art der Norm") which can be "SN or ÄN or ÜN" of the document from the MetadatenDs
+   * block for a single element at a specific date.
+   *
+   * @param eid the eId of the single element
+   * @param date the specific date of the time boundary.
+   * @return "Art der Norm" or empty if it doesn't exist.
+   */
+  public Optional<String> getSingleElementArtDerNorm(final String eid, final LocalDate date) {
```

Comment:
Should we just maybe overload the old `getArtDerNorm` method and not have a new name?

Reply:
Wonderful idea!